gamemaster/game.php

```diff
@@ -309,7 +309,7 @@ public static function create($variantID, $name, $password, $bet, $potType, $pha
 
 		/*
 		 * The password is not salted, because it's given out to several people anyway and it
-		 * isn't worth changing the existing behaviour.
+		 * isn't worth changing the existing behavior.

```

Comment:
Haha don't change Kestas' British spelling.

Reply:
oh hahaha, that was the auto-corrector plugin that I have in VSCode lol